internal/backend/auth/authtokens/authjwttokens/jwt.go

```diff
@@ -18,46 +18,68 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-const issuer = "ak"
+const (
+	issuer       = "https://api.autokitteh.cloud"
+	oneYear      = 365 * 24 * time.Hour
+	devPublicKey = `-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEpn+ugYKvxyjH1PP5M+TyI9AhxnWP
+3NtBzkt35Ppv9aX2YoBMXbbQcgUFDCwKY3QpsCmUILh3vno97lHkbgwjbQ==
+-----END PUBLIC KEY-----`
+	devPrivateKey = `-----BEGIN EC PRIVATE KEY-----
```

Comment:
You can't use hard-coded default private keys, even in a dev server. It makes the dev server absolutely insecure, to the point I can't even run it on my work laptop.

Either way, you need to document in Docusaurus (https://github.com/autokitteh/docs.autokitteh.com/tree/main/docs/config) how to create and configure these keys, regardless of the server's mode, but especially for people who want to run in non-dev mode. Otherwise it's making AK unusable.